cli/dev.ts

```diff
@@ -10,13 +10,15 @@ if the <dir> is empty, the current directory will be used.
 
 Options:
     -p, --port      <port>       A port number to start the Aleph.js app, default is 8080
+        --hostname  <hostname>   The address at which the server is to be started
     -L, --log-level <log-level>  Set log level [possible values: debug, info]
     -r, --reload                 Reload source code cache
     -h, --help                   Prints help message
 `
 
 export default async function (workingDir: string, flags: Record<string, any>) {
   const app = new Application(workingDir, 'development', Boolean(flags.r || flags.reload))
+  const hostname = getFlag(flags, ['hostname'], '0.0.0.0')
   const port = parsePortNumber(getFlag(flags, ['p', 'port'], '8080'))
-  await serve({ app, port })
+  await serve({ app, port,hostname })
```

Comment:
`, hostname`. Please use VSCode because we have certain settings for the formatting

Reply:
i have already fix the format